pyterrier/__init__.py

```diff
@@ -1,2 +1,7 @@
 from .http import http_result, view_result, Ok, NotFound, NoContent as http
 from .pyterrier import PyTerrier
+
+def init(*args, **kawrgs):
+  raise NotImplementedError("You have installed pyterrier the web framework, not PyTerrier the information retrieval toolkit - " +
```

Comment:
I have a limit of 88 chars per line (default of black formatter).

You can perhaps change to something like this:
```python
def init(*args, **kwargs):
    raise NotImplementedError(
        (
            "You have installed pyterrier the web framework, not PyTerrier the "
            "information retrieval toolkit - To install the latter, use "
            "'pip install python-terrier'. For more information see "
            "https://github.com/terrier-org/pyterrier"
            "/wiki/Installation-of-Wrong-Package"
        )
    )
```
This way you don't need to concat the strings using `+` sign and you can keep the line to 88 chars.

Reply:
Another thing, the indentation seems a bit off after the `def`, only 2 spaces.

Reply:
Sorry, should have written offline, rather than in Github editor. now fixed.

Reply:
No worries. It still very long line 100+ chars. I will merge it anyway and I can fix that before I do a release. Would that be ok?

Reply:
> No worries. It still very long line 100+ chars.

Sorry, I seemed to have missed that comment during editing. 
> I will merge it anyway and I can fix that before I do a release. Would that be ok?

Sure, your package, your styling. We appreciate your help.

Reply:
Cool, I will merge. I will add the fix today and release a new version on PyPI tonite. :)